templates/shop/cart/complete/_payByLink.html.twig

```diff
@@ -0,0 +1,58 @@
+{% set payment = order.lastCartPayment() %}
+
+{# TODO remove comments and move style and js to different file (how to) #}
+{% if cw_tpay_get_gateway_config_value(payment.method.gatewayConfig, 'type') == 'pay-by-link' %}
+<style>
+    .bank-tile {
+        display: inline-block;
+        padding: 10px;
+        margin: 10px;
+        border: 1px solid #ccc;
+        cursor: pointer;
+        width: 100px;
+        text-align: center;
+    }
+    .bank-tile img {
+        max-width: 100%;
+        height: auto;
+    }
+    .bank-tile.selected {
+        border-color: blue;
+        background-color: #f0f8ff;
+    }
+
+    .bank-container {
+        display: grid;
+        grid-template-columns: repeat(4, 1fr);
+        gap: 15px;
+    }
+</style>
```

Comment:
Create the `/assets/shop/scss/style.scss` and put the content there. Next, add the
```diff
+import './scss/style.scss';
```

to the `/assets/shop/entrypoint.js`